politeiawww/api/www/v1/v1.go

```diff
@@ -111,6 +109,10 @@ const (
 	// for the routes that return lists of proposals
 	ProposalListPageSize = 20
 
+	// ProposalIndexMimeType is the mime type that an index file should
+	// have
+	ProposalIndexMimeType = "text/plain; charset=utf-8"
```

Comment:
Should be `PolicyIndexFileMimeType` so that its consistent with the other index file policies.  It should also be listed with the other index file policies.